src/TableOfContents/index.tsx

```diff
@@ -138,6 +138,7 @@ const TableOfContentsItem: React.FunctionComponent<ITableOfContentsItem> = ({ it
       <div className={cn('-ml-px', className)}>
         {item.icon && <Icon className="mr-3" icon={item.icon} iconSize={12} />}
         <span className="TableOfContentsItem__name flex-1 truncate">{item.name}</span>
+        {item.meta && <span className="text-sm text-left text-gray font-medium">v{item.meta}</span>}
```

Comment:
The meta property doesn't necessarily have to be a version so let's drop the "v"

```suggestion
        {item.meta && <span className="text-sm text-left text-gray font-medium">{item.meta}</span>}
```